run/CCCX.X-M-V1/RAMSIN.INCUS.V1

```diff
@@ -0,0 +1,855 @@
+   HUCMFILE = '../etc/HUCM-SBM',
+   NDTCOLL = 3,              !Collision-coalescence called once 
+                             !every NDTCOLL time steps
+
+! Aerosol parameters         ! See mic_init.f90 for setting profiles
+                             ! ------------------------------------------
+   IAEROSOL = 1,             ! CCN,GCCN: 0=off, 1=userset
+   ISALT    = 0,             ! Sea Salt: 0=off, 1=userset, 2=saltmodel
+   IDUST    = 0,             ! Dust: 0=off, 1=userset 2=dustmodel
+   IABCARB = 0,              ! Absorbing carbon: 0=off, 1=userset
+
+   IDUSTLOFT = 0,            ! 0 = idealized lofting
+                             ! 1 = Ginoux(2001) erodible fraction data
+
+                             !Dust source file name (used if IDUST=2)
+                             !For IDUSTLOFT=1,read DustErodeFrac-Ginoux2001.txt
+   DUSTFILE = '../etc/DustErodeFrac-Ginoux2001.txt',
+
+   ICCNLEV = 2,              ! Aerosol treatment level
+                             ! 0 = no source or sink
+                             ! 1 = nucleation + deposition sink
+                             ! 2 = same as 1 + evaporation regeneration
+
+   IIFN = 2,                 ! Ice Nuclei treatment level
+                             ! 0 = no ice nuclei
+                             ! 1 = Meyers formula (CIN_MAX)
+                             ! 2 = DeMott(2010) (CIN_MAX)
+                             ! 3 = DeMott(2010) (aerosols D>0.5 microns)
+
+   IIFN_FORMULA = 2,         ! DeMott IN Formulation if IIFN = 2 or 3
+                             ! 1 = DeMott(2010) (composite aerosol types)
+                             ! 2 = DeMott(2014) (dust formula)
+
+   IAERORAD  = 1,            ! Aerosol-Radiation Model: 0 = Off, 1 = On
+
+   IAERODEP  = 1,            ! Aerosol wet and dry deposition
+                             ! (track scavenged aerosols if iccnlev>=2)
+                             ! 0 = Off, 1 = On
+
+                             ! Aerosol initial maximum concentrations
+                             ! userset option. Default vertical profiles:
+                             ! MAXNUMBER*exp(-zt(k)/7000.)
+                             ! ------------------------------------------
+   IAEROPRNT  = 1,           ! Print aerosol init profiles (0=no,1=yes)
+   IAEROHIST  = 0,           ! Re-initialize aerosols on hist restart:0,1 !CHANGEME 
+   ICCN_PROF  = 1,           ! CCN Profile Type
+                             ! 1 = Exponential, using default 7000 m scale height
+                             ! 2 = Constant in the Boundary Layer, Exponential above with a 3500 m scale height
+                             ! 3 = Constant in the Boundary Layer, Linear Decreasing to Free Tropospheric Value,
+                             !     then Linear Decrease to Model Top
+
+   CCN_MAX1   = 2500.,       ! CCN at surface (#/mg) !CHANGEME 500. or 2500. (for Low/High)
+   CCN_MAX2   = 0.,          ! CCN-mode-2 (#/mg) 
```

Comment:
These should be 
CCN1_MAX 
CCN2_MAX
according to bin.rams/RAMSIN.testrunonly

Reply:
Done!